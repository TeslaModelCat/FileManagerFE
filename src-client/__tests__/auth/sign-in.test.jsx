import React from 'react';
import { mount } from 'enzyme';
import SignIn from '../../components/auth/sign-in';

describe('rendering', () => {
  it('renders correctly', () => {
    const tree = mount(<SignIn />);

    expect(tree).toMatchSnapshot();
  });
});
