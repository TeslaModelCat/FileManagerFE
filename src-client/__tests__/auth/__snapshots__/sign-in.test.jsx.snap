// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders correctly 1`] = `
<SignIn>
  <div
    className="card auth-form-body"
  >
    <div
      className="card-body"
    >
      <form
        onSubmit={[Function]}
      >
        <div
          className="form-group"
        >
          <label
            htmlFor="email"
          >
            Email
          </label>
          <input
            aria-describedby="email"
            className="form-control "
            id="email"
            onChange={[Function]}
            placeholder="Enter email"
            type="email"
            value=""
          />
        </div>
        <div
          className="form-group"
        >
          <label
            htmlFor="password"
          >
            Password
          </label>
          <input
            className="form-control "
            id="password"
            onChange={[Function]}
            placeholder="Enter password"
            type="password"
            value=""
          />
        </div>
        <div
          aria-label=""
          className="btn-group"
          role="group"
        >
          <button
            className="btn btn-primary"
            type="submit"
          >
            Sign In
          </button>
        </div>
      </form>
    </div>
  </div>
</SignIn>
`;
